tenable_io/api/sc_containers.py

```diff
@@ -0,0 +1,28 @@
+from tenable_io.api.base import BaseApi
+from tenable_io.api.models import ScContainer
+
+
+class ScContainersApi(BaseApi):
+
+    def delete(self, repository_name, sha256):
+        """Delete a container image.
+
+        :param repository_name: The registry/path of the image. For example, if you have a repository named unix and an
+            image named alpine, you would type unix/alpine.
+        :param sha256: SHA256 checksum of the container image.
+        :raise TenableIOApiException:  When API error is encountered.
+        :return: True if successful.
+        """
+        uri = 'container-security/api/v1/container/%(repository_name)s/manifests/%%(sha256)s' % \
```

Comment:
This is actually correct, but I meant to leave a comment to clarify this, which I've forgotten, so thanks for pointing it out.

The reason it's done like this is that the service strangely does not expect `responsitory_name` with slashes `/` properly encoded, which the `delete` method would do. Therefore, I needed to construct that part of the URI first prior to passing it to the `delete` method.